src/wp-includes/class-wp-http-curl.php

```diff
@@ -77,6 +77,9 @@ public function request( $url, $args = array() ) {
 			'headers'     => array(),
 			'body'        => null,
 			'cookies'     => array(),
+			'decompress'  => false,
+			'stream'      => null,
+			'filename'    => '',
```

Comment:
Comparing the default values to [`WP_Http::request()` found here](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/class-wp-http.php#L207-L211):

```
			'decompress'  => true,
			'stream'      => false,
			'filename'    => null,
```

I'm curious what the actual default values here should be.